src/main/create-operator-json/operators/kali.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+import counterType from '../core/counterTypeEnums';
+import operatorId from '../core/operatorIdEnum';
+import Operator from '../core/operatorFactory';
+import r6operators from "r6operators";
+
+let kali = new Operator(r6operators.kali, operatorId.kali, "Operation Shifting Tides");
+
+kali.addCounterNode(operatorId.goyo, counterType.hard, 'Kali can destroy Goyos shields from the other side.');
```

Comment:
A few style things, would you be able to do apply these to all of the relations you are adding, please?

- Use the fuller name for each gadget e.g. Kali's CSRX 300 Rifle, Kali's Explosive Lances, Goyo's Volcán Shield etc.
- Apostrophes for each operator. e.g. Goyo's

Reply:
fixed